src/components/navbar/navbarElements.js

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components';
+import { NavLink as Link } from 'react-router-dom';
+  
+export const Nav = styled.nav`
+  background: #EF5350;
+  height: 85px;
```

Comment:
You should not use absolute measures on CSS variables, but relative ones. Use percentages or em/rem instead